src/main/java/seedu/address/storage/JsonAdaptedPatient.java

```diff
@@ -81,6 +82,7 @@ public JsonAdaptedPatient(Patient source) {
      * @throws IllegalValueException if there were any data constraints violated in the adapted patient.
      */
     public Patient toModelType() throws IllegalValueException {
+        System.out.println(this.assignedDepartment);
```

Comment:
Is this line necessary? Or was it just for debugging HAHA

Reply:
You are right this is for debugging. Removed it!